src/utilities/toast-manager.test.ts

```diff
@@ -1,3 +1,92 @@
+import faker from "faker";
+import { ToastManager } from "./toast-manager";
+
 describe("ToastManager", () => {
-    test.skip("TODO: Backfill tests for issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/34", () => {});
+    test("when error toast created, returns toastId", () => {
+        // Arrange
+        const testToastId = faker.random.number(99);
+        const toastContent = faker.random.word();
+        const toastOptions = { toastId: testToastId };
+
+        // Act
+        const toastId = ToastManager.error(toastContent, toastOptions);
+
+        // Assert
+        expect(toastId).toBe(testToastId);
+    });
+
+    test("when info toast created, returns toastId", () => {
+        // Arrange
+        const testToastId = faker.random.number(99);
+        const toastContent = faker.random.word();
+        const toastOptions = { toastId: testToastId };
```

Comment:
It should return an id of the created toast without explicitly passing one in, right? Can we add test(s) for those cases?